src/main/java/app/knock/api/model/KnockTenant.java

```diff
@@ -0,0 +1,40 @@
+package app.knock.api.model;
+
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Singular;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+@Value
+@Jacksonized
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KnockTenant {
```

Comment:
Nit: we can call this just `Tenant` -- we only did `KnockMessage` because of namespace collisions with java std lib